src/main/java/com/team4/testingsystem/controllers/TestsController.java

```diff
@@ -54,19 +57,24 @@ public TestsController(TestsService testsService,
 
     @ApiOperation(value = "Get all tests assigned to the current user")
     @GetMapping(path = "/")
-    public List<TestInfo> getCurrentUserTests() {
-        return convertToTestInfoDTO(testsService.getByUserId(JwtTokenUtil.extractUserDetails().getId()));
+    public List<TestInfo> getCurrentUserTests(@RequestParam int pageNumb,
+                                              @RequestParam int pageSize) {
```

Comment:
What do you think about some default values?

Reply:
on the pages there is a choice of the number of lines that will be displayed, I think that the front will send us the amount that they need